plugins/dynamix.docker.manager/include/CreateDocker.php

```diff
@@ -618,7 +618,7 @@ function getUsedPorts() {
             if (empty($arrConfig['Display']) || preg_match("/^Path\s\d/", $arrConfig['Name'])) {
```

Comment:
Yep, because of the mods I made on line 309. Ronald added "Host" prefixing all 1.0 converted fields in JS, so I ported them to PHP.